test/bench/spec/bsonBench.ts

```
@@ -1,77 +1,105 @@
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'serialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
 // Add flat bson decoding
 suite.task({
   documentPath: join(DOCUMENT_ROOT, 'flat_bson.json'),
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'deserialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
 // Add deep bson encoding
 suite.task({
   documentPath: join(DOCUMENT_ROOT, 'deep_bson.json'),
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'serialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
 // Add deep bson decoding
 suite.task({
   documentPath: join(DOCUMENT_ROOT, 'deep_bson.json'),
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'deserialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
 // Add full bson encoding
 suite.task({
   documentPath: join(DOCUMENT_ROOT, 'full_bson.json'),
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'serialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
 // Add full bson decoding
 suite.task({
   documentPath: join(DOCUMENT_ROOT, 'full_bson.json'),
-  library: LIBRARY_SPEC,
-  warmup: 1000,
-  iterations: 10_000,
+  library,
+  warmup,
+  iterations,
   operation: 'deserialize',
-  options: {}
+  options: {},
+  tags: [ALERT_TAG]
 });
 
-suite.run().then(
-  () => {
-    const results = suite.results.map(result => {
+suite
+  .run()
+  .then(() => {
+    return readFile(join(__dirname, '..', '..', 'etc', 'cpuBaseline.json'), 'utf8');
+  }, console.error)
+  .then(cpuBaseline => {
+    if (!cpuBaseline) throw new Error('could not find cpu baseline');
```

Ah related to the other comment, if we just let the error throw then we'll know why it's missing maybe? (probably file not found, but who knows!)